homeassistant/components/connectedcars/device_tracker.py

```diff
@@ -0,0 +1,130 @@
+"""Support for reading vehicle status from ConnectedCars.io."""
+from typing import Any, Callable, Dict, List
```

Comment:
This PR contains 2 platforms. Please remove this one for now, and let's start with the sensors.
After this PR is merged, adding more platforms becomes easier and quicker.

<https://developers.home-assistant.io/docs/creating_component_code_review#5-make-your-pull-request-as-small-as-possible>

Reply:
Well, ok - If that is what it takes, then I will remove it for now and add it in a later PR again